.github/workflows/build.yml

```diff
@@ -103,7 +103,9 @@ jobs:
     - name: Install Python dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install -r requirements.txt
+        mkdir _static
```

Comment:
Do we want this here to avoid the warning:
`WARNING: html_static_path entry '_static' does not exist`